src/mediaplayer.cs

```diff
@@ -261,15 +269,26 @@ interface MPMediaItemArtwork {
 		[Export ("imageWithSize:")]
 		[return: NullAllowed]
 		UIImage ImageWithSize (CGSize size);
+#else
+		[Export ("initWithBoundsSize:requestHandler:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (CGSize boundsSize, Func<CGSize, NSImage> requestHandler);
+
+		[Export ("imageWithSize:")]
+		[return: NullAllowed]
+		NSImage ImageWithSize (CGSize size);
```

Comment:
hmm... that API already exists in `!MONOMAC` so the `#if` should be moved if all platform support this API (and the name can't be changed to something better)